server/src/main/java/org/apache/iotdb/db/mpp/plan/scheduler/FragmentInstanceDispatcherImpl.java

```diff
@@ -210,6 +265,29 @@ private void dispatchRemote(FragmentInstance instance, TEndPoint endPoint)
     }
   }
 
+  public void dispatchRemoteAsync(
```

Comment:
Can we define a seperate class to handle the whole logic

Reply:
Add AsyncPlanNodeSender class.